doc/source/missing_data.rst

```diff
@@ -330,6 +330,10 @@ Interpolation
 
   The ``limit_direction`` keyword argument was added.
 
+.. versionadded:: 0.21.0
+
```

Comment:
Hi Jeff,

The examples for both limit_direction and limit_area are below in the "interpolation limits" sub-section.

I'm mostly trying to get the correct style from inference, so I basically reproduced what had been done in the past for limit_direction.

There is a location (.. _missing_data.interp_limits:) below these versionadded references to which both limit_direction and limit_area can be linked if that is the right style.

Honestly, since version added is part of the docstrings, I'm not sure it needs to be reproduced here at all, but again, that is a bigger style question above my pay grade.  :-)